README.md

```diff
@@ -9,97 +9,117 @@ Issues on Github will be disabled soon.**
 
 # Contributing
 
-- [Styling and formatting your contribution](#styling)
+If this is your first contribution to an open source software (OSS) project, congratulations! Contributing to the docs is a great way to get started; you can learn more about DC/OS, and contribute back to an OSS project right away. No expertise necessary. If you are an old hand, welcome back!
+
+Below you will find some instructions on how to contribute to the DC/OS docs. By following them you help make sure that no one unnecessarily duplicates work, and that the people who can merge (approve and integrate) your contribution are aware that you’re working on something. In order to follow these instructions you’ll need to have a [GitHub account](https://github.com/join?source=header-home) and an account on the [DC/OS JIRA](https://dcosjira.atlassian.net/admin/users/sign-up). If you aren’t signed up for those things yet go do it now. We’ll be here when you get back.
+
 - [Making your contribution](#making)
+- [Styling and formatting your contribution](#styling)
 - [Building and testing your content locally](#test-local)
-- [Submitting your pull request](#submitting)
 
```

Comment:
The workflow feels out of order to me: I'd like to know how to make a contribution before I want to know how that contribution should be styled. Submitting a pull request feels like part of "making your contribution" since you haven't really contributed until your PR is merged.